build.gradle

```diff
@@ -209,18 +146,74 @@ subprojects {
     publications {
       mavenArtifacts(MavenPublication) {
         from components.java
+
         afterEvaluate {
-          pom mavenProjectDescription(project)
           groupId = project.group
           artifactId = project.name
+
+          pom {
+            name = project.mavenName ?: project.name
+            description = project.description ?: project.name
+            url = 'https://github.com/metafacture/metafacture-core'
+            inceptionYear = '2011'
+            developers {
+              developer {
+                id = 'mgeipel'
+                name = 'Markus M. Geipel'
+                email = 'm.geipel@dnb.de'
+                url = 'https://github.com/mgeipel'
+                roles = ['architect', 'developer']
+              }
+              developer {
+                id = 'cboehme'
+                name = 'Christoph Böhme'
+                email = 'c.boehme@dnb.de'
+                url = 'https://github.com/cboehme'
+                roles = ['architect', 'developer']
+              }
+            }
+            organization {
+              name = 'Metafacture'
+              url = 'https://github.com/metafacture'
+            }
+            licenses {
+              license {
+                name = 'The Apache License, Version 2.0'
+                url = 'http://www.apache.org/licenses/LICENSE-2.0.txt'
+              }
+            }
+            mailingLists {
+              mailingList {
+                name = 'Metafacture Mailing List'
+                post = 'metafacture@lists.dnb.de'
+                subscribe = 'metafacture-subscribe@lists.dnb.de'
+                unsubscribe = 'metafacture-unsubscribe@lists.dnb.de'
+                archive = 'http://lists.dnb.de/pipermail/metafacture/'
+              }
+            }
+            scm {
+              connection = 'scm:git:https://github.com/metafacture/metafacture-core.git'
+              developerConnection = 'scm:git:https://github.com/metafacture/metafacture-core.git'
+              url = 'https://github.com/metafacture/metafacture-core'
+              tag = rootProject.scmInfo.tag ?: 'HEAD'
+            }
+            issueManagement {
+              system = 'Github'
+              url = 'https://github.com/metafacture/metafacture-core/issues'
+            }
+            ciManagement {
+              system = 'Github Actions'
+              url = 'https://github.com/metafacture/metafacture-core/actions'
+            }
+          }
         }
       }
     }
     repositories {
       maven {
         name = 'localEmbedded'
         // run generated gradle task `./gradlew
-        // publishMavenArtifactsPublicationToLocalEmbeddedRepository`
+        // publishAllPublicationsToLocalEmbeddedRepository`
         // to publish all subprojects into the same local embedded repo:
         url = "file://${rootDir}/build/repo"
       }
```

Comment:
BTW: What's the purpose of this target? Do we (still) need it?

Reply:
I think we don't need it.

Reply:
Okay, removed in 585c22c.